app/controllers/helpers/TransportSectionHelper.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.helpers
+
+import forms.declaration.ModeOfTransportCode
+import forms.declaration.ModeOfTransportCode.{FixedTransportInstallations, PostalConsignment}
+import forms.declaration.additionaldeclarationtype.AdditionalDeclarationType._
+
+object TransportSectionHelper {
+
+  val altAdditionalTypesOnTransportSection = List(STANDARD_PRE_LODGED, STANDARD_FRONTIER, SUPPLEMENTARY_SIMPLIFIED)
```

Comment:
Not sure what the 'alt' bit stands for in this var name (alternative?). Are these not the 'additionalDecTypesAllowedOnTransportSection'? :thinking:

Reply:
I admit I was not creative enough, but it's tricky to find the right one :grin: 
Even `additionalDecTypesAllowedOnTransportSection` is not correct because it gives you the idea that these are the only ADTs allowed in the transport section, which is not true. It's just that under them there are specific conditions to take into account to correctly address the journey flow.
And yes, `alt` stands for alternative, which if you ask, I can use in full :slightly_smiling_face:

Reply:
What do you think of maybe `additionalDecTypesAllowedOnInlandOrBorder` (as these are the only types allowed on this page by the `JourneyAction` refiner)? I understand they are also used elsewhere for other decision points but this seems like the strongest candidate IMO :slightly_smiling_face:  

If not then maybe some sort of comment explaining their significance/purpose?